src/test/java/persistence/sql/ddl/EntityTest.java

```diff
@@ -0,0 +1,58 @@
+package persistence.sql.ddl;
+
+import org.junit.jupiter.api.Test;
+import persistence.sql.ddl.entity.*;
+import persistence.sql.ddl.exception.IncorrectIdFieldException;
+import persistence.sql.ddl.exception.NotEntityException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class EntityTest {
+    @Test
+    public void 클래스를_분석하여_Entity로_변경한다() {
```

Comment:
```suggestion
class EntityTest {
    @Test
    void 클래스를_분석하여_Entity로_변경한다() {
```
junit 5부터는 테스트 클래스 및 메소드에 pulibc을 제거해주셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods